service/core/client/src/main/scala/com/lightbend/lagom/internal/client/CircuitBreakersPanelInternal.scala

```diff
@@ -56,6 +56,18 @@ private[lagom] class CircuitBreakersPanelInternal(
 
   private val createCircuitBreaker = new JFunction[String, Option[CircuitBreakerHolder]] {
 
+    private val allExceptionAsFailure: Try[_] => Boolean = {
+      case _: Success[_] ⇒ false
+      case _             ⇒ true
+    }
+
+    private def failureDefinition(whitelist: Set[String]): Try[_] => Boolean = {
+      case _: Success[_] ⇒ false
+      case Failure(t) if whitelist.contains(t.getClass.getName) ⇒ false
+      case _ ⇒ true
+    }
```

Comment:
This will create a function tailored by the user so whitelisted exceptions are not opening the CB.